plugins/quantum_k_means/schemas.py

```diff
@@ -74,6 +88,27 @@ class InputParametersSchema(FrontendFormBaseSchema):
             "input_type": "select",
         },
     )
+    tol = ma.fields.Float(
+        required=True,
+        allow_none=False,
+        metadata={
+            "label": "Tolerance for Convergence Criteria",
+            "description": "The tolerance is given in percentage, i.e. an input of 5 = 5%.\n"
+            "The algorithm does multiple iterations and after each iteration it checks how the cluster assignments for our data points "
+            "have changed. If the input tolerance is 5%, then the algorithm stops, if less than 5% of the "
+            "assignments have changed.",
+        },
+    )
+    max_runs = ma.fields.Integer(
+        required=True,
+        allow_none=False,
+        metadata={
+            "label": "Maximum Number of Iterations",
+            "description": "The algorithms does multiple iterations. After reaching the maximum number of iterations, "
+            "the algorithm terminates, even if the tolerance isn't reached.",
+            "input_type": "text",
```

Comment:
Because browsers can act differently when they know what type of input is expected in an input field. For numbers they prevent text inputs. This provides a better user experience as the user gest immediate feedback from the browser on what kind of input is allowed and does not have to rely on the validate button to check inputs.
<https://developer.mozilla.org/en-US/docs/Learn/Forms/HTML5_input_types#numeric_field>